web/views/index.html.erb

```diff
@@ -18,7 +18,12 @@
       <tbody>
         <% @inhibitors.each do |inhibitor| %>
           <tr id="antidote-inhibitor-<%= CGI.escape(inhibitor.id) %>">
-            <td><%= inhibitor.treatment %></td>
+            <td width="25%">
+              <%= inhibitor.treatment %>
+              <% if inhibitor.treatment == "suspend" %>
+                <em class="text-muted" style="margin-left: 10px"> Jobs suspended: <%= inhibitor.suspension_group_size %> </em>
+              <% end %>
```

Comment:
Let's move this to a separate column, before Actions:

```
| treatment | qualifier | statistics | actions |
```

I think all inhibitor types will benefit from having some metrics on how many jobs were inhibited.

Reply:
In fact, instead of having `suspensions` list we may have `inhibited-jobs` list that will allow check how many jobs were inhibited and which they were, before we delete them all.

Reply:
Correction. We don't want to have inhibited jobs list. If user selected skip - that means they don't want the jobs to be performed, e.g. because they pollute the queue. All we need to do is to create stats page for which jobs classes were inhibited and how often. Inspiration should be taken from Sidekiq's own metrics :D

Reply:
for suspended jobs that page will also who list of currently suspended jobs

Reply:
do we want to add another page for these stats, or should we add it to this page. My concern with another page is that it may add too much noise to sidekiq ui

Reply:
I've been thinking about this for a while now. And metrics/stats are unrelated to this PR. So, if we will have stats, it might be nice to have an inhibitor-dedicated page where we will show some metrics (like which jobs were inhibited and how much).

We don't need full-blown metrics. Something as simple as total count will do the trick.
Same page might be used to show suspended jobs as well though...